XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/HandPointer.cs

```diff
@@ -1,65 +1,41 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) XRTK. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System;
-using UnityEngine;
-using XRTK.Definitions.Utilities;
-using XRTK.Interfaces.InputSystem.Controllers.Hands;
-using XRTK.Services;
-
 namespace XRTK.SDK.UX.Pointers
 {
     /// <summary>
     /// Pointer class for hand controller pointers.
     /// </summary>
     public class HandPointer : BaseControllerPointer
     {
-        private IMixedRealityHandRay handRay;
-
-        /// <inheritdoc />
-        protected override void OnEnable()
-        {
-            base.OnEnable();
-
-            if (handRay == null)
-            {
-                InitializeHandRay();
-            }
-        }
-
-        /// <inheritdoc />
-        protected override void Start()
-        {
-            base.Start();
-
-            if (handRay == null)
-            {
-                InitializeHandRay();
-            }
-        }
-
-        /// <inheritdoc />
-        public override bool TryGetPointerPosition(out Vector3 position)
-        {
-            return base.TryGetPointerPosition(out position);
-        }
-
-        /// <inheritdoc />
-        public override bool TryGetPointerRotation(out Quaternion rotation)
-        {
-            return base.TryGetPointerRotation(out rotation);
-        }
-
-        /// <inheritdoc />
-        public override bool TryGetPointingRay(out Ray pointingRay)
-        {
-            return base.TryGetPointingRay(out pointingRay);
-        }
-
-        private void InitializeHandRay()
-        {
-            SystemType handRayType = MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.HandTrackingProfile.HandRayType;
-            handRay = (IMixedRealityHandRay)Activator.CreateInstance(handRayType.Type);
-        }
+        //private IMixedRealityHandRay handRay;
```

Comment:
The handray was simply removed?

Reply:
Not sure why Stephen removed this. This was work in progress code and will most likely come back.

Reply:
Yeah I was just trying to get it to compile. Likely will come back but again I'm not a fan of having a different ray type for hands. Imho there should be just one type. I'm fine with having an interface I suppose but this is likely cruft left over from porting from mrtk.